blueprints/addon/index.js

```diff
@@ -52,7 +55,16 @@ module.exports = {
 
     contents.name = this.project.name();
 
-    fs.writeFileSync(path.join(this.path, 'files', 'bower.json'), JSON.stringify(contents, null, 2));
+    this._writeContentsToFile(contents, 'bower.json');
+  },
+
+  _writeContentsToFile: function(contents, fileName) {
+    fs.writeFileSync(path.join(this.path, 'files', fileName), JSON.stringify(contents, null, 2) + '\n');
+  },
+
+  _alphabetizeDependencies: function(contents) {
```

Comment:
this method doesnt use the classes internal state, it should not be on the prototype.